scala/scala.bzl

```diff
@@ -464,15 +464,15 @@ def _collect_jars(dep_targets, dependency_analyzer_is_off = True):
     else:
       return _collect_jars_when_dependency_analyzer_is_on(dep_targets)
 
-def is_dependency_analyzer_off(ctx):
-  if not hasattr(ctx.attr, 'dependency_analyzer_mode_soon_to_be_removed'):
+def is_dependency_analyzer_on(ctx):
+  if (hasattr(ctx.attr,"_dependency_analyzer_plugin")
+    #when the strict deps FT is removed this will be removed as well
```

Comment:
I think this comment is a bit confusing.
The FT is this line (line 471)...
Unless I am missing something which other line will be deleted when FT will be removed?